code/game/machinery/hologram.dm

```diff
@@ -59,10 +59,37 @@ Possible to do for anyone motivated enough:
 	var/static/list/holopads = list()
 	var/obj/effect/overlay/holoray/ray
 	var/offset = FALSE
+	var/on_network = TRUE
+
+/obj/machinery/holopad/tutorial
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	flags_1 = NODECONSTRUCT_1
+	on_network = FALSE
+	
+/obj/machinery/holopad/tutorial/Initialize(mapload)
+	. = ..()
+	if(!mapload)
+		return
+	var/obj/item/disk/holodisk/new_disk = locate(/obj/item/disk/holodisk) in src.loc
+	if(new_disk && !disk)
+		new_disk.forceMove(src)
+		disk = new_disk
 
+/obj/machinery/holopad/tutorial/attack_hand(mob/user)
+	if(!istype(user))
+		return
+	if(user.incapacitated() || !is_operational())
+		return
+	if(replay_mode)
+		replay_stop()
+	else if(disk && disk.record)
+		replay_start()
+	return

```

Comment:
trailing return

Reply:
Intended, it should block all further down actions so that interact and what not do not get called, the tutorial one only allows the use of on and off.

Reply:
The return literally does nothing. If you want to call the parent function you have to use `..()`. Not sure what you mean by that comment, exactly.

Reply:
oh whoops you're right fuck